### PR TITLE
Empty the default of TrailSequences in NukePower

### LINKED ISSUE #16449: Warning: Custom sprite image of actor is null
Travis is reporting a collection of
```
OpenRA.Utility(1,1): Warning: Custom sprite image of actor tmpl is null
```
warnings on bleed in both RA and TD. Regression from a recent PR.

### PR DESCRIPTION
**Pull request by abcdefg30**
Closes #16449. (Which was a regression from #16399.)

### CODE REVIEW

**Comment by pchote** [comment]
```
Exception of type `System.ArgumentException`: Collection must not be empty.
Parameter name: ts
  at OpenRA.Exts.Random[T] (System.Collections.Generic.IEnumerable`1[T] ts, OpenRA.Support.MersenneTwister r, System.Boolean throws) [0x0001e] in <c8b954f08ec945deb9a52aac105872c2>:0 
  at OpenRA.Exts.Random[T] (System.Collections.Generic.IEnumerable`1[T] ts, OpenRA.Support.MersenneTwister r) [0x00000] in <c8b954f08ec945deb9a52aac105872c2>:0 
  at OpenRA.Mods.Common.Effects.NukeLaunch+<>c__DisplayClass27_1.<Tick>b__0 (OpenRA.World w) [0x00000] in <f7894682d433497f960a12939f763177>:0 
```

When launching the cluster missile in TS

File: OpenRA.Mods.Common/Traits/SupportPowers/NukePower.cs [hunk 1]
```diff
@@ -52,7 +52,7 @@ class NukePowerInfo : SupportPowerInfo, IRulesetLoaded, Requires<BodyOrientation
 		public readonly string TrailImage = null;
 
 		[Desc("Loop a randomly chosen sequence of TrailImage from this list while this projectile is moving.")]
-		[SequenceReference("TrailImage")] public readonly string[] TrailSequences = { "idle" };
+		[SequenceReference("TrailImage")] public readonly string[] TrailSequences = { };
```
Comment (pchote) [inline OpenRA.Mods.Common/Traits/SupportPowers/NukePower.cs]: NukeLaunch assumes that this is not empty if `TrailImage` is defined.
Can you please add a `TraitSequences.Any()` check there or throw an exception from `RulesetLoaded` below?

Comment (abcdefg30) [reply OpenRA.Mods.Common/Traits/SupportPowers/NukePower.cs]: Done.

**Comment by abcdefg30** [comment]
Updated.

**Review by pchote — APPROVED** [review]

**Comment by reaperrr** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/ceb81bf03a8484d853f43a40c418a28ef3a6ca76...684b0096ff262e323ea7c61ffe3aafa8b8f8a171

**Pull request merged**